code/modules/martial_arts/krav_maga.dm

```diff
@@ -3,6 +3,17 @@
 	var/datum/action/neck_chop/neckchop = new/datum/action/neck_chop()
 	var/datum/action/leg_sweep/legsweep = new/datum/action/leg_sweep()
 	var/datum/action/lung_punch/lungpunch = new/datum/action/lung_punch()
+	var/datum/action/neutral_stance/neutral = new/datum/action/neutral_stance()
+
+/datum/action/neutral_stance
+	name = "Neutral Stance - You relax, cancelling your last Krav Maga stance attack."
+	button_icon_state = "neutralstance"
+
+/datum/action/neutral_stance/Trigger()
+	to_chat(owner, "<b><i>You cancel your prepared attack.</i></b>")
+	owner.visible_message("<span class='danger'> [owner] relaxes his stance.")
```

Comment:
```suggestion
	owner.visible_message("<span class='danger'>[owner] relaxes [owner.p_their()] stance.</span>", "<span class='notice'>You cancel your prepared attack.</span>")
```

In HTML, any opening tag needs a closing tag. Also changed `his` to the owner's gender pronoun.
Finally, `visible_message` allows a second argument equal to the message displayed to the owner only -- no need to do a separate `to_chat` call.